examples/gltf/app.js

```diff
@@ -99,7 +99,9 @@ export class DemoApp {
 
     this.glOptions = {
       // Use to test gltf with webgl 1.0 and 2.0
-      webgl2: true
+      webgl2: true,
+      // alpha causes issues with some glTF demos
+      alpha: false
```

Comment:
We might want to add some information about this in the glTF or PBRMaterial docs?